en-US/ranking/criteria.md

```diff
@@ -20,8 +38,22 @@ The following pertains to metadata that your maps contain such as song artists,
 * **The artist and title must be absolutely correct.**
 * **The artist and title must be romanized.** However, non-romanized characters are allowed to be used in source and tags.
 * **The artist, title and source must not repeat in tags.** However, romanizing or translating the source in tags is encouraged, in case it's relevant.
+* **Source must not a website name. It should be filled with album, mixtape, game name, etc.** However, if it comes from youtube, soundcloud and such, source should remain empty.
```

Comment:
```suggestion
* **The source should be an album, mixtape, game, show, etc. that the song comes from.** Youtube, Soundcloud, etc., are not valid sources. This field may be left empty if there is no relevant source.
```

Reply:
im not against this but kind of ehhhhhhh, the vagueness is ok but you know, most people (that still confused to rc) still did it. youtube, soundcloud, bandcamp etc are valid, but not good to be added on source field. would you suggest another wording for me?

Reply:
idk this seems decently clear to me